packages/kbn-http-tools/src/create_server.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { Server, ServerOptions } from '@hapi/hapi';
+import { ListenerOptions } from './get_listener_options';
+
+export function createServer(serverOptions: ServerOptions, listenerOptions: ListenerOptions) {
+  const server = new Server(serverOptions);
+
+  server.listener.keepAliveTimeout = listenerOptions.keepaliveTimeout;
```

Comment:
These are plain copies from `src/core/server/http/http_tools.ts`. I just created distinct files for each method in the `@kbn/http-tools` package.